packages/block-editor/src/components/list-view/block.js

```diff
@@ -245,17 +237,13 @@ function ListViewBlock( {
 			id={ `list-view-block-${ clientId }` }
 			data-block={ clientId }
 			isExpanded={ canExpand ? isExpanded : undefined }
-			aria-selected={ !! isSelected || forceSelectionContentLock }
```

Comment:
Why is `aria-selected` no longer needed? Is it that it only needs to be set once at the cell level?

Reply:
Yes, looks that way. It did nothing when I only tested it on the `tr`.